### ISSUE TITLE
Sort numerically

### ISSUE THREAD

**Issue by tk1** [aslagle/reactive-table#178]
When sorting columns, they will be sorted alphabetically. There shoud be an option for sorting numerically, or still better, the possiblity of giving a comparison functon.

**Comment by aslagle** [comment]
I agree, a comparison function would be useful, but I think the columns do sort numerically if the values are numbers. There's a sortByValue option for fields to sort by the original value instead of the output of fn.

**Comment by ph0rn** [comment]
I can confirm columns don't sort by value if they're all numbers. For example 100 sorts lower than 2.

**Comment by mrmowgli** [comment]
+1 for a comparison function sort.

**Comment by pmacom** [comment]
+1
